CHANGES/.TEMPLATE.rst

```diff
@@ -0,0 +1,36 @@
+{# TOWNCRIER TEMPLATE #}
```

Comment:
I think, this folder also needs a separate README explaining what to do. GitHub will then render it in the directory view.

Reply:
The idea is good; the README is not mandatory though.
aiohttp doesn't have it; we have instructions in CONTRIBUTING document.
I think it can work for multidict as well.

Reply:
I saw it work here well: https://github.com/tox-dev/tox/tree/master/docs/changelog.

Reply:
If you want to add README here -- you are welcome. In a separate PR.